package.json

```diff
@@ -11,7 +11,7 @@
   },
   "engines": {
     "yarn": "^1.15.2",
-    "node": ">=16.14.0"
+    "node": ">=14.17.0"
```

Comment:
Husky won't let any commits through unless Node is set to 14.17 or later, as Husky doesn't seem to respect the Node version offered via nvm.

Reply:
hmmm this isn't a change we'll want to merge lets get your local env fixed up together at office hours today if we can!